src/main/java/com/plesba/datamanager/target/CSVTarget.java

```diff
@@ -62,7 +64,7 @@ public void processDataFromInputStream() throws IOException {
                 if (streamByte != 10) { //end of line
                     recordStringBuffer.append((char) streamByte);
                 } else { //process record
-                    recordCount++;
+                                                   recordCount++;
```

Comment:
That's some interesting indentation. Does your IDE have an automatic formatting function?